src/UI/Dependency/Dummy/DummyFeatureManager.ts

```diff
@@ -1,4 +1,4 @@
-import { FeatureManager, JsonParserId, StatusLicense } from "@/Core";
+import { FeatureManager, JsonParserId, ServerStatus, StatusLicense } from "@/Core";
 
 export class DummyFeatureManager implements FeatureManager {
```

Comment:
what's the difference between the MockFeatureManager, and DummyFeatureManager?

Reply:
Dummies in our app are  to indicate that given hook/provider is not set up and they will throw error, I assume it's a way to faster indicate that for test setup something is missing in the flow.

MockFeature Manager is, well, Mocked data